src/m365/base/ContextCommand.ts

```diff
@@ -0,0 +1,37 @@
+import * as fs from 'fs';
+import AnonymousCommand from './AnonymousCommand';
+import { Logger } from '../../cli/Logger';
+import { M365RcJson } from './M365RcJson';
+
+export default abstract class ContextCommand extends AnonymousCommand {
+  public saveContextInfo(contextInfo: any, logger: Logger): Promise<any> {
```

Comment:
Let's rename the `contextInfo` arg to `context` to avoid confusion with SharePoint's context info